### DISCUSSION TITLE
ui.log in footer width

### DISCUSSION THREAD

**Discussion by mbolder** [zauberzeug/nicegui#1472] [Q&A] [answered]
### Question

Hello,

I have a log and logout button in my footer, and would like to set the size of log to 80%.

I tried the following, but what ever size I select apart from w-full is ignored. 

```py
with ui.footer():
    log = ui.log(max_lines=100).classes('w-8/10')
    ui.button(on_click=lambda: (app.storage.user.clear(), ui.open('/login')), icon='logout',color="red").props('outline round').classes('self-end w-2/10')
```

**Comment by falkoschindler** [comment, marked as answer]
Hi @mbolder,

When using Tailwind classes for layouting, you either need to use their classes (e.g. `10/12`, see https://tailwindcss.com/docs/width) or specify a custom width like so: `w-[80%]`. But the class `w-8/10` does not exist.

**Comment by mbolder** [comment]
Thank you,
I thought their fractions were just examples. w-[80%] works just fine.
